pyproject.toml

```diff
@@ -176,7 +176,7 @@ max-args = 6
 notice-rgx = """
 # This code is a Qiskit project.
 #
-# \\(C\\) Copyright IBM 2024\\.
+# \\(C\\) Copyright IBM \\d{4}((,\\s)\\d{4})*\\.
```

Comment:
This is an ad-hoc fix. We should have a more general discussion on how to handle copyright headers in the Qiskit addons.

Here, I did what I also used to do back in the application modules (which is to ensure the year of the latest change of a file is present with at most 2 years being listed). However, this regex cannot enforce that pattern consistently. In the application modules we had a [tool](https://github.com/qiskit-community/qiskit-nature/blob/main/tools/check_copyright.py) to check these headers. We could consider using something similar but rather than copying this tool everywhere I would rather do something similar to the `extremal-python-dependencies` tool and ship this separately (if we decide that this is needed).